### PR TITLE
gh-125 Add Popover Component to Examples page

### PR DESCRIPTION
**Pull request by oleksii-yurchenko**
- install popper.js
- create example popover component

### CODE REVIEW

File: apps/web-portal-client/lib/features/examples/components/TippyExample.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react'
+import Tippy from '@tippy.js/react'
+import 'tippy.js/dist/tippy.css'
```
Comment (nesterone) [inline apps/web-portal-client/lib/features/examples/components/TippyExample.jsx]: I think in our case it makes sense to choose the Tippy library, it's much easier to use

However, we need to decide on where we would include `css` dependencies, I mean the one place to do that.

**Review by nesterone — CHANGES_REQUESTED** [review]

**Comment by oleksii-yurchenko** [comment]
close #125

File: apps/web-portal-client/lib/components/Popover.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+  )
+
+  return (
+    <Tippy
+      content={content}
+      visible={visible}
+      arrow
+      animation="scale"
+      duration={0}
```
Comment (nesterone) [inline apps/web-portal-client/lib/components/Popover.jsx]: Another way to write it is  to use spread for props 
<Tippy {...props} content={content} />
```
Popover.propTypes = {
  ...Tippy.propTypes,
  children: childrenPropTypes,
}

```

but we can leave as it is right now, let's just define  `duration` and `delay` in propTypes and defaultProps

**Review by nesterone — CHANGES_REQUESTED** [review]
It looks good !

**Review by nesterone — APPROVED** [review]

**Pull request merged**